### PR TITLE
 JM example [JM-1]

### PR DESCRIPTION
**Pull request by jm220**
* **Please check if the PR fulfills these requirements**
- [X ] The pull request title contains the ticket number in the title. For example [CB-1]

* **What kind of change does this PR introduce?** (Bug fix, new feature...)

This adds three new routes on a new folder titled JM 

* **What is the current behavior?What does the code do before you made the new changes?** *

Current behavior - Index is empty
New Changes - new index file under JM folder

* **What is the new behavior?**
Creates an API that can be tested with Postman

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR? Run a new command?)

N/A

* **Other information**:

### CODE REVIEW

File: JM/index.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+            postmanresponse.json({"message": "Error "})
+        })
+    console.log(data)
+})
+
+app.delete('/deletePerson', function(postmanRequest, postmanresponse){
+    const id = postmanRequest.query['id']
+    console.log('Deleting ID: ' + id )
+    axios.delete('https://java-sample-api-2020.herokuapp.com/deletePerson?id=' + id) 
+    .then(function(response){
```
Comment (arsoljon) [inline JM/index.js:35]: Possible refactoring would be to move the 'route + id' into a separate variable to improve readability.

**Review by arsoljon — DISMISSED** [review]
Works and looks amazingly.

File: JM/index.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+const app = express()
+const axios = require('axios')
+app.use(express.json())
+
+app.post('/addPerson', function(postmanRequest, postmanresponse){
+    console.log(postmanRequest.body)
+    axios.post('http://java-sample-api-2020.herokuapp.com/addPerson', postmanRequest.body) 
+    .then(function(apiResopnse){
+        postmanresponse.json({"message": "User added"})
+    })
```
Comment (Codubee) [inline JM/index.js:11]: can you add an error handler here

**Comment by Codubee** [comment]
Please fill out the PR template

File: JM/index.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+})
+
+app.get('/getAllPeople', function(postmanRequest, postmanresponse){
+    const url = 'http://java-sample-api-2020.herokuapp.com/getAllPeople'
+    data = axios.get(url)
```
Comment (Codubee) [inline JM/index.js]: I think that the data variable here would be equal to a promise. Id recommend looking at and understanding the [example get route](https://github.com/Codubee/Node-Js-API-Examples/blob/axios-examples/index.js)

**Review by Codubee — CHANGES_REQUESTED** [review]
Left some comments

**Review by jm220 — COMMENTED** [review]
Made changes

File: JM/index.js (hunk 3 above)
Comment (Codubee) [reply JM/index.js]: I believe that this is still incorrect Jaime. Lets setup some time so I can explain what is going on

File: JM/index.js [hunk 4]
```diff
@@ -17,10 +17,10 @@ app.post('/addPerson', function(postmanRequest, postmanresponse){
 
 app.get('/getAllPeople', function(postmanRequest, postmanresponse){
     const url = 'http://java-sample-api-2020.herokuapp.com/getAllPeople'
-    data = axios.get(url)
+    axios.get(url)
         .then((response) => {
             const data = response.data
```
Comment (arsoljon) [inline JM/index.js]: Everything looks good to me. Line 22 has an unused variable 'data'. Consider removing or using.

**Review by arsoljon — DISMISSED** [review]
Left 1 comment, code looks good.

**Review by arsoljon — APPROVED** [review]
Looks good to me.

**Review by Codubee — APPROVED** [review]
LGTM!

**Pull request merged**